repos/system_upgrade/el7toel8/actors/pammodulesscanner/actor.py

```diff
@@ -0,0 +1,37 @@
+from leapp.actors import Actor
+from leapp.models import PamConfiguration, PamService
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+from leapp.libraries.common.pam import PAM
+
+from os import listdir
+from os.path import isfile, islink, join
```

Comment:
please prefer to import only os
```suggestion
import os
```